handbook/engineering/README.md

```diff
@@ -669,20 +672,27 @@ After it's in a release formally, the bug should be treated like any other piece
 #### Fast track for Fleeties
 Fleeties do not have to wait for QA to reproduce the bug. If you're confident it's reproducible, it's a bug, and the reproduction steps are well-documented, it can be moved directly to the reproduced state.
 
-#### Release testing
-When a release is in testing, QA should use the the Slack channel #help-qa to keep everyone aware of issues found. All bugs found should be reported in the channel after creating the bug first.
+### Release testing
+When a release is in testing, QA should use the Slack channel #help-qa to keep everyone aware of issues found. All bugs found should be reported in the channel after creating the bug first.
 
 In the #help-qa channel, product may decide whether the bug is a release blocker. When a release-blocking bug is found, product is responsible for communicating the delay to company stakeholders. 
 
 Release blockers include:
-1. Critical bugs (defined below)
-2. New functionality that impacts previous stable functionality
-3. Incomplete features as defined or implied in the specs
+1. Release-blocking bugs (defined below)
+2. Critical bugs (defined below)
+3. New functionality that impacts previous stable functionality
+4. Incomplete features as defined or implied in the specs
```

Comment:
These are examples of release blockers and should be in the description section. otherwise i would expect them to have descriptions below

Reply:
Good catch, updated.